lib/router/router.js

```diff
@@ -62,6 +62,11 @@ p.getInfo = function(req) {
     || routeInfo.urlInfo.query.cache !== 'false' // or request isn't disabling cache
   ;
 
+  // Append search query param to pathname if pathDelimiter = '?'
+  if (this.options.pathDelimiter === '?') {
+    routeInfo.urlInfo.pathname = `${routeInfo.urlInfo.pathname}${routeInfo.urlInfo.search}`;
```

Comment:
In order for the split to work, isn't the `pathDelimiter` also needed? `pathname` would not include it right?

Reply:
In other words, shouldn't this be:

```suggestion
    routeInfo.urlInfo.pathname = `${routeInfo.urlInfo.pathname}?${routeInfo.urlInfo.search}`;
```